src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Configuration/ConfigUpdaterStep.cs

```diff
@@ -54,7 +57,7 @@ public ConfigUpdaterStep(IEnumerable<IUpdater<ConfigFile>> configUpdaters, IOpti
                 throw new ArgumentNullException(nameof(configUpdaterOptions));
             }
 
-            _configFilePaths = configUpdaterOptions.Value?.ConfigFilePaths ?? Array.Empty<string>();
+            _configFilePaths = configUpdaterOptions.Value.SelectMany(s => s.ConfigFilePaths).ToArray();
```

Comment:
Nit-pick - we may want a .Distinct in case different extensions add the same config file.